src/DynamoCore/Logging/DynamoAnalyticsClient.cs

```diff
@@ -96,6 +97,10 @@ class Dummy : IDisposable
 
         public virtual IAnalyticsSession Session { get; private set; }
 
+        private DateTime LastMachineHBLogTime;
+        private DateTime LastUserHBLogTime;
```

Comment:
since I don't see this set anywhere initially, won't it always be null and no events will ever be sent?

Reply:
`DateTime` types are initiated with a default value just like booleans

Reply:
ah it's a struct